.github/workflows/format-code.yml

```diff
@@ -0,0 +1,61 @@
+name: "Format code"
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  format:
+    name: "Format code"
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/format')
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v2
+      - name: "Format Markdown code"
+        run: |
+          # Download the pull request's data
+          PULL_REQUEST="/tmp/pr.json"
+          PULL_REQUEST_URL=$(jq -r ".issue.pull_request.url" "$GITHUB_EVENT_PATH")
+          curl --url $PULL_REQUEST_URL --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' -o "$PULL_REQUEST"
+
+          BASE_REPO_FULL_NAME=$(jq -r ".base.repo.full_name" "$PULL_REQUEST")
+          HEAD_REPO_FULL_NAME=$(jq -r ".head.repo.full_name" "$PULL_REQUEST")
+          HEAD_REF=$(jq -r ".head.ref" "$PULL_REQUEST")
+
+          # We cannot push commits to forks
+          if [ "$BASE_REPO_FULL_NAME" != "$HEAD_REPO_FULL_NAME" ]; then
+            echo "Cannot format code on a fork"
+            exit 1
+          fi
```

Comment:
It just came up on Slack. This may actually work with the default token because technically the action is executed in the scope of this repo, not the forked repo. I'm not certain and the docs don't cover this case well, so you'd have to try it. Sorry for not thinking of that before when you asked about forks 😬